pkg/controllers/user/cis/clusterScanHandler.go

```diff
@@ -53,23 +63,54 @@ func (csh *cisScanHandler) Create(cs *v3.ClusterScan) (runtime.Object, error) {
 	if !v3.ClusterScanConditionCreated.IsTrue(cs) {
 		logrus.Infof("cisScanHandler: Create: deploying helm chart")
 
+		skipOverride := false
 		appInfo := &appInfo{
 			appName:     cs.Name,
 			clusterName: cs.Spec.ClusterID,
 		}
 		if cs.Spec.ScanConfig.CisScanConfig != nil {
-			appInfo.skip = strings.Join(cs.Spec.ScanConfig.CisScanConfig.Skip, ",")
 			if cs.Spec.ScanConfig.CisScanConfig.DebugMaster {
 				appInfo.debugMaster = "true"
 			}
 			if cs.Spec.ScanConfig.CisScanConfig.DebugWorker {
 				appInfo.debugWorker = "true"
 			}
+			if cs.Spec.ScanConfig.CisScanConfig.Skip != "" {
+				skipOverride = true
+			}
 		}
-		// Check if the configmap is populated
-		cm, err := csh.userCtxCMLister.Get(v3.DefaultNamespaceForCis, v3.ConfigMapNameForUserConfig)
-		if err != nil && !kerrors.IsNotFound(err) {
-			return cs, fmt.Errorf("cisScanHandler: Create: error fetching configmap %v: %v", err, v3.ConfigMapNameForUserConfig)
+
+		var cm *v1.ConfigMap
+		if skipOverride {
+			// create the cm
+			cm = &v1.ConfigMap{
+				TypeMeta: metav1.TypeMeta{
+					Kind: "ConfigMap",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name: getOverrideConfigMapName(cs),
+				},
+				Data: map[string]string{
+					ConfigFileName: cs.Spec.ScanConfig.CisScanConfig.Skip,
+				},
+			}
+			logrus.Infof("cisScanHandler: Create: creating skip override configmap %v with contents: %v", cm.Name, cs.Spec.ScanConfig.CisScanConfig.Skip)
```

Comment:
is it safe to print the map contents in info? Can there be any sensitive information?

Reply:
nope, there is no sensitive info. Only the test numbers which are being skipped.